Makefile

```
@@ -0,0 +1,24 @@
+.PHONY: generate clean reset
+.ONESHELL:
+
+SHELL := /bin/zsh
+
+generate:
```

checkmake (0.2.2)

[warning] Target body for "generate" exceeds allowed length of 5 (6).

(maxbodylength)